### ISSUE TITLE
CTRL+A in the search field (and other dialog text fields) should select the text in the text input

### ISSUE THREAD

**Issue by bziemons** [istoph/editor#14]
Similar to how CTRL+A works across most of the programs I use, I would expect it to select all text in the currently focused text field.

**Comment by textshell** [comment]
Yes, the ZInputBox widget is currently missing selection support.
I've opened tuiwidgets/tuiwidgets#3 for tracking the needed work in tui widgets. Likely this only works well if also tuiwidgets/tuiwidgets#4 is implemented.
